### DISCUSSION TITLE
Form Field Manager custom HTML not displaying result of math properly

### DISCUSSION THREAD

**Discussion by erikacft** [impress-org/givewp#6634] [Developer Q&A]
We are attempting to add a field in a Legacy Form using Form Field Manager HTML that will display a number based on the donation amount divided by 10. 

We've tried this a number of ways and can get the label to display, but the calculated number does not display, it shows as a weird number picker. 

Any help you can provide that would point us in the right direction would be great. In simple terms, all we want it to do is take the donation total (ex. $100) and divide it by a number we choose (10) and display the result (10) next to our chosen label.

Here's an example of the code we tried that created the weird pick box:

&lt; div>
 &lt; label for="total-donation-divided-by-10">Total Donation / 10:</label>
  &lt; input type="number" id="total-donation-divided-by-10" value="{
    var total = {amount} / 10;
    return total;
  }" style="color: black; font-size: 12pt;">
&lt; /div>

It shows the label, but instead of showing just a number (even as text would be fine) there is the box with arrows that can change the number up or down. We just want it to display the result of a basic calculation. 

Thank you in advance for your insights!

**Comment by kjohnson** [comment]
The increment/decrement UI is a result of it being a `type="number"` input element which is added by the browser. To display a calculated value I would suggest using a `<div />`.

**Comment by kjohnson** [comment]
@erikacft is the calculated value working, otherwise?

**Reply by erikacft** [reply]
Thank you for your quick suggestion. It's hard to tell because even when we use <div nothing shows up, only the label. It's blank underneath.

**Reply by erikacft** [reply]
Hi again, we tried this code without the number type. It displays the label and below it shows plain text "{ var total = {amount} / 10; return total; }" but it doesnt actually perform the calculation or display the result. 

&lt; div>
&lt;label for="total-donation-divided-by-10">Total Donation / 10:</label>
&lt;div id="total-donation-divided-by-10" style="color: black; font-size: 12pt;">
   {
    var total = {amount} / 10;
   return total;
   }
  &lt;/div>
&lt;/div>
